exonum/src/api/public/system.rs

```diff
@@ -12,15 +12,20 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+use std::time::SystemTime;
+use std::collections::HashMap;
+
+use crypto::PublicKey;
```

Comment:
Module `crypto` is part of Exonum, so it should be in the next section. :wink:

Reply:
Done